scene/2d/remote_transform_2d.cpp

```diff
@@ -141,7 +142,13 @@ NodePath RemoteTransform2D::get_remote_node() const {
 }
 
 void RemoteTransform2D::set_use_global_coordinates(const bool p_enable) {
+	if (use_global_coordinates == p_enable) {
+		return;
+	}
```

Comment:
Probably makes sense to also add such guards to other setters (`set_use_position` etc.).